### DISCUSSION TITLE
Resolve 18 support

### DISCUSSION THREAD

**Discussion by gmiranda** [xtremestuff/protune-transforms#1] [General]
Hello,

Does this work on DaVinci Resolve 18? I've tried following your instructions to no avail. I'm using Linux, though (Pop_OS).

Thanks!

**Comment by RikshaDriver** [comment]
Hi,

Yes it should work. Which Transform are you trying to use and have you ensured the files are installed to the appropriate location?

**Reply by gmiranda** [reply]
The location might be the problem:  I can't find the LUT Folder in the path specified on the README. I'm using ~/.local/share/DaVinciResolve/.LUT instead.

And for the ACES IDT folder: ~/.local/share/DaVinciResolve/ACES Transforms/IDT

**Reply by RikshaDriver** [reply]
I've just checked on Elementary OS and the folder is **/opt/resolve/LUT**

I've updated the documentation as well.

The transforms won't appear if you place them anywhere else.
